src/components/Alert/index.test.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import { render, screen } from '@testing-library/react';
+
+import WarningIcon from 'components/Icon/WarningIcon';
+
+import Alert from '.';
+
+const testAlerData = {
+  dataTestId: 'testAlert',
+  errorMessage: ['test error'],
+  alertHeader: 'Error',
+  icon: 'alert-icon',
+};
+
+describe('Alert', () => {
+  it('render the alert container', () => {
+    render(
+      <Alert
+        Icon={WarningIcon}
+        alertHeader={testAlerData.alertHeader}
+        dataTestId={testAlerData.dataTestId}
+        errorMessage={testAlerData.errorMessage}
+      ></Alert>
+    );
+    const container = screen.getByTestId(testAlerData.dataTestId);
+    expect(container).toBeVisible();
```

Comment:
We should separate the assignment, actions, and expectations:
```suggestion
    render(
      <Alert
        Icon={WarningIcon}
        alertHeader={testAlerData.alertHeader}
        dataTestId={testAlerData.dataTestId}
        errorMessage={testAlerData.errorMessage}
      ></Alert>
    );

    const container = screen.getByTestId(testAlerData.dataTestId);

    expect(container).toBeVisible();
```